<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic Encryption Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QCheckBox" name="show_passphrase_checkbox">
      <property name="text">
       <string>Show Passphrase</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0" colspan="2">
        <widget class="QTextEdit" name="plaintext_textedit"/>
       </item>
       <item row="0" column="1" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="encrypt_button">
         <property name="text">
          <string>Encrypt</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignLeft">
        <widget class="QLabel" name="plaintext_label">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Plaintext</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="decrypt_button">
         <property name="text">
          <string>Decrypt</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignRight">
        <widget class="QLabel" name="ciphertext_label">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ciphertext</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QTextEdit" name="ciphertext_textedit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="passphrase_lineedit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" alignment="Qt::AlignRight">
     <widget class="QLabel" name="passphrase_label">
      <property name="text">
       <string>Passphrase</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
